INSTALL.md

```diff
@@ -36,6 +36,22 @@ make bash
 
 All the source and dependencies for this project will be in place, making it possible to run porta and the tests from inside the container. See [Run Porta](#run-porta)
 
+#### Running the application
+
+If you want to run the application using docker, you can use the docker compose file to start all dependencies.
+
+In the first time, you need to setup the database. To configure it, run: (the Master and User password will be `p`)
+
+```
+MASTER_PASSWORD=p USER_PASSWORD=p make dev-setup
+```
+
+and to start the application and all dependencies:
+
+```
+make up
```

Comment:
Is it `run` here instead?
```
$ make up
make: *** No rule to make target `up'.  Stop.
```

Reply:
yeah!.. you are right. I renamed the `up` to `dev-run`.

Will change it. thanks!

Reply:
@josemigallas fixed it in the doc. Also, renamed it to `dev-start` like we discussed.